contributing_to_docs/doc_guidelines.adoc

```diff
@@ -37,17 +37,17 @@ Every assembly file should contain the following metadata at the top, with no li
 ----
 :_content-type: ASSEMBLY                                        <1>
 [id="<unique-heading-for-assembly>"]                            <2>
-= Assembly title                                                <3>
-include::_attributes/common-attributes.adoc[]                   <4>
+include::_attributes/common-attributes.adoc[]                   <3>
+= Assembly title                                                <4>
 :context: <unique-context-for-assembly>                         <5>
                                                                 <6>
 toc::[]                                                         <7>
 ----
 
 <1> The content type for the file. For assemblies, always use `:_content-type: ASSEMBLY`. Place this attribute before the anchor ID or, if present, the conditional that contains the anchor ID.
-<2> A unique (within OpenShift docs) anchor ID for this assembly. Use lowercase. Example: cli-developer-commands
-<3> Human readable title (notice the `=` top-level header)
-<4> Includes attributes common to OpenShift docs.
+<2> A unique (within OpenShift docs) anchor ID for this assembly. Use lowercase. Example: cli-developer-commands.
+<3> Includes attributes common to OpenShift docs.
```

Comment:
I just noticed this, but should the note be under #3 since it is sort of referring to the attributes not being in the attributes file? I'm sorry I didn't see this earlier, but since #3 and #4 have switched places, I think the note should move up with #3.

Reply:
@kcarmichael08 - fixed! Good spot!